schemes/sima_diagnostics/check_energy_fix_diagnostics.meta

```diff
@@ -0,0 +1,59 @@
+[ccpp-table-properties]
+  name = check_energy_fix_diagnostics
+  type = scheme
+
+[ccpp-arg-table]
+  name  = check_energy_fix_diagnostics_init
+  type  = scheme
+[ errmsg ]
+  standard_name = ccpp_error_message
+  units = none
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+[ errflg ]
+  standard_name = ccpp_error_code
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = out
+
+[ccpp-arg-table]
+  name  = check_energy_fix_diagnostics_run
+  type  = scheme
+[ te_ini_dyn ]
+  standard_name = vertically_integrated_total_energy_using_dycore_energy_formula_at_start_of_physics_timestep
+  units = J m-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = in
+[ te_cur_dyn ]
+  standard_name = vertically_integrated_total_energy_using_dycore_energy_formula
+  units = J m-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ teout ]
+  standard_name = vertically_integrated_total_energy_using_dycore_energy_formula_at_end_of_physics_timestep
+  units = J m-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ eshflx ]
+  standard_name = net_sensible_heat_flux_through_top_and_bottom_of_atmosphere_column
```

Comment:
Just checking if this is the correct standard name, or if it should be `net_sensible_heat_flux_through_top_and_bottom_of_atmosphere_column_from_global_total_energy_correction`, as listed in the `history_add_field` long name?

In other words is this the net heat flux into the atmospheric column, or is this a flux adjustment made by the fixer itself in order to enforce energy conservation?  Just want to make sure everything is being named as accurately as possible.

Reply:
It is from the energy fixer so

```net_sensible_heat_flux_through_top_and_bottom_of_atmosphere_column_from_global_total_energy_correction'''

is more accurate.

Reply:
Thanks @nusbaume and @PeterHjortLauritzen, I agree that `_from_global_total_energy_correction` is more accurate (so it is used in the diagnostics) but setting without the suffix here is intentional so this quantity is passed as an input into `check_energy_chng` for checking the heat flux added by `check_energy_fix`.

It is equivalent to this code in `physpkg.F90`:
```fortran
       call check_energy_cam_fix(state, ptend, nstep, flx_heat)
       call physics_update(state, ptend, ztodt, tend)
       call check_energy_cam_chng(state, tend, "chkengyfix", nstep, ztodt, zero, zero, zero, flx_heat)
```

The `flx_heat` here is the `eshflx`.

Reply:
Hi @jimmielin, just confirming then that there are other situations where `net_sensible_heat_flux_through_top_and_bottom_of_atmosphere_column` is passed into `check_energy_chng`, but that flux is not coming from the energy fixer?  If so then what you did here makes total sense to me.  Thanks!

Reply:
Great, thanks for verifying @jimmielin!  This PR then looks good to me, so I'll go ahead and approve it now.